src/MCPClient/lib/clientScripts/parse_mets_to_db.py

```diff
@@ -66,7 +66,10 @@ def parse_files(root):
             if not format_version:
                 name = amdsec.findtext('.//premis:formatName', namespaces=ns.NSMAP)
                 print('Format name', name)
-                format_version = fpr_models.FormatVersion.active.get(description=name)
+                try:
+                    format_version = fpr_models.FormatVersion.active.get(description=name)
+                except:
```

Comment:
I think that it'd be best to explicitly catch`fpr_models.FormatVersion.MultipleObjectsReturned` only to avoid ignoring other possible exceptions like `fpr_models.FormatVersion.DoestNotExist`?